File: app/src/main/java/com/ybj/myapplication/java/arithmetic/Lesson1_2.java
package com.ybj.myapplication.java.arithmetic;

import com.ybj.myapplication.java.ArithmeticTest;

/**
 * Created by 杨阳洋 on 2020/7/7.
 * 查找数组中最大元素
 */
public class Lesson1_2 implements ArithmeticTest {
    @Override
    public void test() {
        int a [] = {1,4,3};
        int max = -1;
        for (int i = 0 ; i < a.length ; i++){
            if(a[i] > max) {
                max = a[i];
            }
        }
        System.out.println(max);

    }
}

File: app/src/main/java/com/ybj/myapplication/kotlin/arithemtic/Lesson1_2.kt
package com.ybj.myapplication.kotlin.arithemtic

/**
 * Created by 杨阳洋 on 2020/7/7.
 * 查找数组中最大元素
 */
class Lesson1_2 {
    fun test() {
        var a:IntArray = intArrayOf(1,4,3)
        var max :Int = -1
        for (i:Int in a.indices){
            if (a[i] > max) max = a[i]
        }
        println(max)
    }
}
